aries-site/src/tests/navigation/search.js

```diff
@@ -75,3 +75,19 @@ test('should navigate to correct page when user is only using keyboard ', async
     .expect(getLocation())
     .eql(baseUrl + expectedPath);
 });
+
+/*This test will fail until issue #320 is fixed. The page scroll is at the incorrect section of the 
+page therefore it is failing. Going to leave commented out for now. 
+*/
+
+// test('should navigate to correct hash after user clicks a suggestion that leads to a page subsection', async t => {
+//   const page = 'Background Colors';
+//   const backgroundColors = Selector('#background-colors')
+//   const suggestion = getSuggestion(page);
+
+//   await t
+//     .typeText(ReactSelector(Search), 'col')
+//     .click(suggestion)
+//     .expect(backgroundColors.getBoundingClientRectProperty("top"))
+//     .within(-150, 150);
+// });
```

Comment:
why is this block needed?

Reply:
This block tests if the user for example searches background-colors from the home page it should take them to the subsection #background-colors of the page. Currently it takes the user to the top of the page which needs to be fixed from issue #320 I commented out so our tests wont fail since we know what the problem is.

Reply:
If it makes sense to keep this out for now until we fix issue #320 then I can save it locally and add it back in once that is fixed.

Reply:
If we are planning on keeping this comment, add an inline explanation of why it is commented and what required to have it back in.

Reply:
okay I did add 
This test will fail until issue #320 is fixed. The page scroll is at the incorrect section of the 
page therefore it is failing. Going to leave commented out for now. 
 
on line 79-80 not sure if you saw that but I can explain further